src/libopensc/card-openpgp.c

```diff
@@ -2072,9 +2072,11 @@ pgp_set_security_env(sc_card_t *card,
 
 	LOG_FUNC_CALLED(card->ctx);
 
+	/* The SC_SEC_ENV_ALG_PRESENT is set always so let it pass for GNUK */
```

Comment:
Maybe it might make sense to have more explanation in the commit log, e.g., what was previously broken, and why?